src/Header/Header.tsx

```diff
@@ -24,63 +24,138 @@ const Header = () => {
     }
   };
 
-  const linkTextStyle = {
-    visibility: menu ? 'visible' : '',
-    opacity: menu ? '1' : '',
-    transition: 'opacity 200ms',
-    whiteSpace: 'nowrap',
-  } as React.CSSProperties;
+
+
+const width = menu ? '14rem' : '3rem';
+const StyledNav = styled.nav`
+  z-index: 3;
+  height: 100vh;
+  position: fixed;
+  left: 0;
+  background: rgb(24, 21, 21);
+  transition: width 200ms ease;
+  width: ${width};
```

Comment:
See the comment above. It should end up looking something like 
```suggestion
  width: ${({ menuOpen }): StyledNavProps} => menuOpen ? '14rem' : '3rem';
```